Refactor buttons on communities of practice and program areas pages

Zhu-Joseph opened the pull request:
Fixes #2158 

### What changes did you make and why did you make them ?

  - Updated button on **line 69** on communities of practice page ```website/pages/communities-of-practice.html```
  - Updated button on **line 71** on communities of practice page ```website/pages/communities-of-practice.html```
  - Updated button on **line 87** on program areas page ```website/pages/program-areas.html```

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>

![Before](https://user-images.githubusercontent.com/76148970/136678409-e9a029f9-7ff6-44d0-96fd-b5e089f7538d.png)

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![After](https://user-images.githubusercontent.com/76148970/136678412-aae162f4-f128-4665-b5a5-7e47afe12f36.png)

</details>

Zhu-Joseph committed:
Refactor buttons on communities of practice and program areas pages

adam-abundis reviewed (changes requested):
@Zhu-Joseph Thank you for your pull request. My apologies for not being clear in the issue. The button should look the same as before. We need to add css to the specific button elements to make sure they match the before picture. Let me know if you have any questions. Thank you. 🌟 🌟 🌟

Zhu-Joseph committed:
Update CSS for buttons

Zhu-Joseph commented:
> @Zhu-Joseph Thank you for your pull request. My apologies for not being clear in the issue. The button should look the same as before. We need to add css to the specific button elements to make sure they match the before picture. Let me know if you have any questions. Thank you. 🌟 🌟 🌟

@abuna1985 Hi Adam, I updated the code and made a commit. I did have to update the css to get the buttons to match what they looked like before. Below is an image of the updated code. 

<details >
<summary> Visuals Changed </summary>

<br>

![After2](https://user-images.githubusercontent.com/76148970/136896687-d43df33d-9c0f-472d-adde-258f3e5420ed.png)
</details>

adam-abundis commented on _sass/components/_communities-of-practice.scss:
```diff
@@ -126,3 +126,7 @@
   font-size: 14px;
 }
+
+.align-center a {
```
@Zhu-Joseph Remove line 130-132 (`.align-center a` styling). We are going to consolidate the styling.

adam-abundis commented on _sass/components/_program-areas.scss:
```diff
@@ -163,3 +163,7 @@
 margin-bottom: 48px;
 }
+
+.join-us-footer-body a {
```
@Zhu-Joseph Remove 167-169 (`.join-us-footer-body` styling). We are going to consolidate the styling.

adam-abundis reviewed (changes requested):
@Zhu-Joseph My apologies for the continuous edits. This is my last one. Can you:
1. Remove the scss files changes (comments in Files Changed)
2. Edit the `btn--default` class in `_sass/elements/_buttons.scss`

**From**
https://github.com/hackforla/website/blob/481ab1973aac0acad969a8a32c8857f9e1365751/_sass/elements/_buttons.scss#L40

**To**
`display: inline-flex;`

This will consolidate the styling since all our buttons are links now. As soon as this is updated, I will approve and merge.

Thank you!

Zhu-Joseph committed:
Update scss to standardize styling

Zhu-Joseph commented:
> @Zhu-Joseph My apologies for the continuous edits. This is my last one. Can you:
> 
> 1. Remove the scss files changes (comments in Files Changed)
> 2. Edit the `btn--default` class in `_sass/elements/_buttons.scss`
> 
> **From**
> 
> https://github.com/hackforla/website/blob/481ab1973aac0acad969a8a32c8857f9e1365751/_sass/elements/_buttons.scss#L40
> 
> **To** `display: inline-flex;`
> 
> This will consolidate the styling since all our buttons are links now. As soon as this is updated, I will approve and merge.
> 
> Thank you!

Hi Adam, 

I made the following changes:
 - Removed commented code
 - Updated line 40 on ```_sass/elements/_buttons.scss``` to ```display: inline-flex;```

Please let me know if there is anything else you would need me to do!

adam-abundis reviewed (approved):
@Zhu-Joseph Thank you again for making another round of changes and for helping us refactor our buttons. I was able to review the changes and everything looks good to me. I am going to approve and merge this. Keep up the great work! ⭐ ⭐ ⭐
